.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,4 +7,5 @@
 <!-- Provide a brief single sentence for the changelog. -->
 
 ## Status
+- [ ] `make format` run on the codebase (may have to `conda install black isort autoflake -c conda-forge` or `pip install black isort autoflake`)
```

Comment:
I took the liberty of adding this directly to your branch, so feel free to tweak/object.